zebra-rpc/src/methods.rs

```diff
@@ -2702,7 +2704,11 @@ mod opthex {
     }
 }
 /// Returns the proof-of-work difficulty as a multiple of the minimum difficulty.
-pub async fn chain_tip_difficulty<State>(network: Network, mut state: State) -> Result<f64>
+pub async fn chain_tip_difficulty<State>(
+    network: Network,
+    mut state: State,
+    should_use_default: bool,
```

Comment:
This new field is undocumented, and I do not know how to set it. (This method is used downstream in Zaino.)